typescript/src/client/README.md

```diff
@@ -1,27 +1,30 @@
-[SDK](../../../) / [TypeScript](../../) / **Client**
+[Frameworks](../../../) → **Client** → **TypeScript** | Python (soon)
 
 # <img src="https://github.com/superfaceai/sdk/raw/main/docs/logos/client.png" alt="{ }" width="30" height="30" />  Superface Client
 
-Superface Client provides a seamless way to integrate intelligent tools into your applications. It allows you to fetch tool definitions, run tools, and manage user connections.
+> [!NOTE]  
+> This is a low-level client, you will need to handle mapping of function descriptors & tool calling manually. For framework specific use, please refer to [supported frameworks](../../../).
```

Comment:
shouldn't it be function definitions? 

https://platform.openai.com/docs/guides/function-calling#function-definition